### DISCUSSION TITLE
How to simulate keypress event for the webpage or specific html element?

### DISCUSSION THREAD

**Discussion by assopri** [cefsharp/CefSharp#3762] [Q&A]
I want to simulate sending keypress event for the entire webpage or being focused on the particular html element. How can I implement that with Cefsharp winforms 91 version?

Also can I add simulation of modifiers pressing (like Control, Alt and Shift) along with particular key?

**Comment by amaitland** [comment]
See https://stackoverflow.com/questions/44736904/issue-with-cefsharp-browser-sendkeys to get you started

You'll need to make sure the control has focus. You can focus a html element using javascript.

**Reply by assopri** [reply]
Cool! It works just fine! THank you for your job!
